src/MICore/JsonLaunchOptions.cs

```diff
@@ -486,6 +486,12 @@ public partial class PipeTransportOptions
         [JsonProperty("pipeArgs", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public List<string> PipeArgs { get; private set; }
 
+        /// <summary>
+        /// Command line arguments passed to the pipe program to execute a reote command.
```

Comment:
Spelling: `reote` -> `remote`

```suggestion
        /// Command line arguments passed to the pipe program to execute a remote command.
```

Reply:
Use the description from XSD instead: `Arguments passed to the pipe executable to run a remote Unix (ex: bash or similar) shell command. This is used for running 'kill' on the remote system, or other commands that the debugger may need. If not specified, the debugger will do its best without the shell. But some features, such as setting breakpoints in run mode, may not work. This string should contain the string '{0}' which will be replaced with the command to execute.`

---
In reply to: [590604833](https://github.com/microsoft/MIEngine/pull/1111#discussion_r590604833) [](ancestors = 590604833)

Reply:
pipeCmd is to be used in VS. Updated OpenFolderSchema.json.
Fixed comment